### PR TITLE
Docs-nits

### PR DESCRIPTION
**Pull request by mcrascal**
- Removes keyboard shortcut in search box on mobile 
- Enable HMR on core components in docs and example project 
- Fix bug where click to copy hover would clip out of the window
- Add support for markdown prop descriptions to prop listing component, migrates existing props 
- Align dev command in root to other packages `pnpm dev:docs`
- Lint

### CODE REVIEW

File: sites/docs/pages/+layout.svelte [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+</head>
+
+<EvidenceDefaultLayout
+	{data}
+	githubRepo="https://github.com/evidence-dev/evidence"
 	slackCommunity="https://slack.evidence.dev"
 	xProfile="https://twitter.com/evidence_dev"
 	builtWithEvidence
-	maxWidth="1500" 
+	neverShowQueries
```
Comment (archiewood) [inline sites/docs/pages/+layout.svelte:18]: until we have a good proposal for a query / component / source code viewer, I'd advocate for keeping this available?

File: packages/ui/core-components/src/lib/organisms/layout/header/SearchButton.css [hunk 2]
```diff
@@ -1,9 +1,9 @@
 .DocSearch-Button {
-	@apply bg-white hover:bg-gray-50 transition-colors duration-200 hover:text-gray-800 rounded-md flex gap-16 cursor-pointer py-1 pl-2 pr-1 shadow-sm sm:text-xs text-gray-600 border font-sans font-medium  items-center;
+	@apply bg-white hover:bg-gray-50 transition-colors duration-200 hover:text-gray-800 rounded-md flex gap-16 cursor-pointer py-1 pl-2 sm:pr-1  pr-20 shadow-sm sm:text-xs text-gray-600 border font-sans font-medium  items-center;
```
Comment (archiewood) [inline packages/ui/core-components/src/lib/organisms/layout/header/SearchButton.css:2]: when on `xs` size the text displaying `Search` is larger than on `md+` - Seems odd

File: packages/ui/core-components/src/lib/organisms/layout/header/SearchButton.css [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 }
 .DocSearch-Search-Icon {
 	display: none;
```
Comment (archiewood) [inline packages/ui/core-components/src/lib/organisms/layout/header/SearchButton.css:5]: could replace the sm text with just this search icon to save horizontal space?

File: sites/docs/pages/components/big-value.md [hunk 4]
```diff
@@ -1,180 +1,199 @@
----
-title: Big Value
-sidebar_position: 1
```
Comment (archiewood) [inline sites/docs/pages/components/big-value.md:3]: deleting this throws off the sort in the sidebar.

was it intentional to delete all this content below also?

**Review by archiewood — COMMENTED** [review]
Overall like it

Some thoughts for your review

File: sites/docs/pages/components/big-value.md (hunk 4 above)
Comment (mcrascal) [reply sites/docs/pages/components/big-value.md:3]: GPT refactor run amok, change incoming for this one and a couple others.

**Review by ud3sh — APPROVED** [review]

**Pull request merged**